### DISCUSSION TITLE
Error when call a procedure that has a ref_cursor as out parameter

### DISCUSSION THREAD

**Discussion by skysegbr** [oracle/python-oracledb#167] [General]
When I call the proc using strings it works:
`result = cur.var(oracledb.CURSOR)`
`cur.callproc('MANAGEMENT.RET_EMAIL_RECEIVER', ['23', '1', '1', '1', result])`

but when I send using integer it got stuck:
`cur.callproc('MANAGEMENT.RET_EMAIL_RECEIVER', [23, 1, 1, 1 , result])`

The procedure expects a number as in parameter.
Best regards.

**Comment by anthony-tuininga** [comment]
Opened as issue #169
